ontolearner/base/learner.py

```diff
@@ -17,27 +16,29 @@ def train(self, train_data: Any, task: str):
     def predict(self, eval_data: Any, task: str) -> Any:
         pass
 
-    def train_predict(self, train_data:Any, eval_data:Any, task:str) -> Any:
+    def train_predict(self, train_data: Any, eval_data: Any, task: str) -> Any:
         self.train(train_data, task)
         predicts = self.predict(eval_data, task)
         return predicts
 
-class AutoLearnerLLM(ABC):
+
+class AutoLLM(ABC):
     def __init__(self):
         self.model = None
         self.tokenizer = None
 
-    def load(self, model_id:str):
+    def load(self, model_id: str):
         pass
 
     def generate(self, inputs: List[Any]):
```

Comment:
Can we specify the output that this `generate` function will return? is it a `List` or `Dict` or what?